### PR TITLE
Bump oc branch for 1.18.8

### PR DESCRIPTION
**Pull request by damemi**
Previous base pr (1.18.2): https://github.com/openshift/kubernetes/pull/123

Following https://github.com/openshift/enhancements/blob/master/enhancements/oc/upgrade_kubernetes.md

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go [hunk 1]
```diff
@@ -1211,7 +1237,7 @@ func TestGetListComponentStatus(t *testing.T) {
 	cmd.Run(cmd, []string{"componentstatuses"})
 
 	expected := `NAME            STATUS      MESSAGE   ERROR
-servergood      Healthy     ok        
+servergood      Healthy     ok
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go:1240]: Don't

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go [hunk 2]
```diff
@@ -1718,7 +1744,7 @@ func TestGetMultipleTypeTableObjectsWithDirectReference(t *testing.T) {
 service/baz   ClusterIP   <none>       <none>        <none>    <unknown>
 
 NAME       STATUS    ROLES    AGE         VERSION
-node/foo   Unknown   <none>   <unknown>   
+node/foo   Unknown   <none>   <unknown>
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go:1747]: Nope

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go (hunk 1 above)
Comment (damemi) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go:1240]: Dropping these spaces was originally added to 1.18.2 in https://github.com/openshift/kubernetes-kubectl/pull/21, should we re-add them?

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**